frontend/src/metabase/components/Popover/Popover.module.css

```diff
@@ -179,79 +112,51 @@
   z-index: 1;
 }
 
-:global(.PopoverBody--withBackground .PopoverFooterWhenIsNotEqualOps),
-:global(.popover .PopoverFooterWhenIsNotEqualOps) {
-  bottom: 0;
-  padding-bottom: 12px;
-  padding-top: 10px;
-  width: calc(100% - 26px);
-}
-
-:global(.PopoverParameterFieldWidgetFooter) {
-  position: fixed;
-  bottom: -7px;
-}
-
+/* Global tether (tippy.js dependency) classes */
```

Comment:
We don't migrate global classes for libraries since we don't control those classes.